component/attr_cache/attr_cache.go

```diff
@@ -580,8 +580,6 @@ func (ac *AttrCache) Chmod(options internal.ChmodOptions) error {
 
 // Chown : Update the file with its new owner and group (when datalake chown is implemented)
 func (ac *AttrCache) Chown(options internal.ChownOptions) error {
-	log.Trace("AttrCache::Chown : Change owner of file/directory %s", options.Name)
-
 	err := ac.NextComponent().Chown(options)
```

Comment:
We shall update the metadata to store user and group supplied as input for the file if remote call was successful